src/main/java/core/basesyntax/service/date/DateConverterImpl.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.date;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DateConverterImpl implements DateConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> rawData) {
```

Comment:
The method convertToTransaction() does not convert dates, it converts a list of strings to a list of FruitTransaction objects. The class and method names should reflect their purpose. It may be better to move this method to a class named `TransactionConverter` or similar.